src/auth/jwtclient.ts

```diff
@@ -261,4 +268,19 @@ export class JWT extends OAuth2Client {
     }
     this.apiKey = apiKey;
   }
+
+  /**
+   * Using the key or keyFile on the JWT client, obtain an object that contains
+   * the key and the client email.
+   */
+  async getCredentials() {
```

Comment:
I'd like public methods to have their return type (and param types) explicitly specified.

Reply:
👍